tests/flask/test.py

```diff
@@ -0,0 +1,49 @@
+from application import app
```

Comment:
Not a big deal for test file, but we generally put `from` lines below `import` after a blank line.

Reply:
Ok will fix.